src/assets/main.scss

```diff
@@ -1,6 +1,7 @@
 @use 'bulma/sass/utilities/derived-variables' with (
   $primary: hsl(261.6, 51.87%, 47.25%),
-  $link: hsl(261.6, 51.87%, 47.25%)
+  $link: hsl(261.6, 51.87%, 47.25%),
+  $warning: hsl(0, 52%, 47%)
```

Comment:
red is `danger`, `warning` is yellow.
Anyway, made some small aesthetic changes - moved this to dropdown.